server/user/user.model.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
+const Audience = require('../shared/auth/audience');
+const bcrypt = require('bcrypt');
+const { callbackify } = require('util');
```

Comment:
@ExtensionEngine/tailor Are we ok with this? It sets minimal required version of node to `v8.2.0`: https://nodejs.org/api/util.html#util_util_callbackify_original

/cc @kjuej